docusaurus.config.js

```diff
@@ -100,11 +100,7 @@ const config = {
           title: "Resources",
           items: [
             {
-              label: "Get Started",
-              href: "https://www.stellar.org/start",
-            },
-            {
-              label: "Learn",
+              label: "Stellar Learn",
```

Comment:
Probably don't need this? We should link to the Stellar homepage somewhere but probably in the header not the footer